Python/crossfire/crossfire/occurrences.py

```diff
@@ -12,9 +12,10 @@ def __init__(self, client, id_state, id_cities=None, limit=None, format=None):
         self.next_page = 1
         self.yielded = 0
 
-        self.params = {"idState": id_state}
+        self.params = {"idState": id_state, "page": self.next_page}
```

Comment:
Não precisamos configurar a página aqui, isso é responsabilidade do método que carrega as ocorrências.

```suggestion
        self.params = {"idState": id_state}
```

Reply:
Uh.... ok. Perfeito. 
Com isso o primeiro request vai sem o parâmetro `page`. Os demais com `page=2`. 
Por isso tive que atualizar os testes.